amundsen_application/static/js/components/TableDetail/EditableSection/index.tsx

```diff
@@ -42,17 +45,27 @@ export class EditableSection extends React.Component<EditableSectionProps, Edita
         setEditMode: this.setEditMode,
       });
     });
-
+    if (!this.props.readOnly) {
```

Comment:
Notice how the only real difference here is whether or not the button is shown, so we can do:
```
return (
      <section className="editable-section">
        <div className="section-title title-3">
          { this.props.title }
          {
            !this.props.readOnly &&
            <button className={"btn btn-flat-icon edit-button" + (this.state.isEditing? " active": "")} onClick={ this.toggleEdit }>
              <img className={"icon icon-small icon-edit" + (this.state.isEditing? " icon-color" : "")} />
            </button>
           }
        </div>
        { childrenWithProps }
      </section>
    );
````

Reply:
ah ok thanks for pointing that out! fixing